t/lib/OpenQA/Test/Utils.pm

```diff
@@ -93,7 +93,7 @@ sub cache_minion_worker {
             require OpenQA::CacheService;
             local $ENV{MOJO_MODE} = 'test';
             note('Starting cache minion worker');
-            OpenQA::CacheService::run(qw(run));
+            OpenQA::CacheService::run(qw(run --dequeue-timeout 1));
```

Comment:
wow, what is that? Can we speedup everything by factor 10 by making this `.1`? :) You could go into more details in the git commit message. That would be nice.

Reply:
Unfortunately there's a point where performance gets worse again (implementation detail upstream...). 😁

The Minion worker is designed as a daemon with a blocking mainloop that primarily blocks when waiting for new jobs to be dequeued. By default `Minion::Backend::SQLite` blocks for 5 seconds before handling control back to `Minion::Worker`, where it does housekeeping stuff like reaping failed jobs and so on before blocking to dequeue again. That option limits the time it blocks to 1 second. So our workers can stop 4 seconds earlier on average, and this test stops a lot of workers.

Reply:
A `0` timeout might work, but is a little risky since the mainloop could start eating a lot of cpu. 🤔

Reply:
But at the cost of almost 5 times the cpu cycles. I think `1` has the best balance between runtime and cpu cost.

Reply:
The option `--dequeue-timeout` itself comes from [Minion::Command::worker](https://metacpan.org/pod/Minion::Command::minion::worker#SYNOPSIS), which `::run()` calls indirectly.